SourceCode/Power/ftt_p_main.py

```diff
@@ -805,11 +803,25 @@ def solve(data, time_lag, iter_lag, titles, histend, year, domain):
                             data_dt['BCET'][:, tech, c2ti['8 std ($/MWh)']] 
                             * (1.0 + data['BCET'][:, tech, c2ti['16 Learning exp']] * dw[tech]/data['MEWW'][0, tech, 0]))
             
+            sector_coupling_assumps = get_sector_coupling_dict(data, titles)
+            
+            battery_lifetime = 12 #assuming 12 years
+            
+            time_steps = 4 #per quarter in a year
+            
+            capacity_batteries_current_timestep = data["MSSC"] * sector_coupling_assumps["GW to GWh"]
+            capacity_batteries_last_timestep = data_dt["MSSC"] * sector_coupling_assumps["GW to GWh"]
+            
+            quarterly_cap_additions = capacity_batteries_current_timestep - capacity_batteries_last_timestep
+            
+            cumulative_quarterly_capacity_power = quarterly_cap_additions + (capacity_batteries_current_timestep/(battery_lifetime/time_steps))
```

Comment:
You are now dividing by (12/4), i.e. by 3. Instead, we want to divide by 12 and 4, so divide by 48. Intuitively, there is less end-of-life replacement each quarter than each year.